pages/1.12/release-notes/1.12.1/index.md

```diff
@@ -0,0 +1,169 @@
+
+### Mesosphere Jupyter Service (MJS)
+- Delivered secure, [cloud-native Jupyter](https://docs.mesosphere.com/services/beta-jupyter/) Notebooks-as-a-Service to empower data scientists to perform analytics and distributed machine learning on elastic GPU-pools with access to big and fast data services.
+- Secured connectivity to data lakes and data sets on S3 and (Kerberized) HDFS.
+- GPU-enabled Spark and distributed TensorFlow.
+- OpenID connect authentication and authorization with support for Windows Integrated Authentication (WIA) and Active Directory Federation Services (ADFS)
+
+### Observability - Metrics
+- Introduced a flexible and configurable metrics pipeline with multiple output formats.
+- Enhanced support for application metric types including histograms, counters, timers, and gauges.
+- Support for sample rates and multi-metrics packets. 
+- Mesos framework metrics are now [available](http://mesos.apache.org/documentation/latest/monitoring/#frameworks).
+- No longer require modifications when collecting metrics via Prometheus endpoint in 1.11.
+
+[enterprise]
+### Private Package Registry
+[/enterprise]
+- Enabled [on-premise package distribution and management](https://docs.mesosphere.com/1.12/administering-clusters/repo/package-registry/).
+- Enabled air-gapped Virtual Private Cloud package management.
+- Simplifies package artifact management.
+- Package-specific controls for adding/removing/updating packages within a cluster.
+- Package management CLI.
+
+### Installation and Upgrade
+- Fully support installing and operating a cluster on SELinux hardened OS with SE Linux in targeted-enforcing mode for all hardened non-DC/OS components.
+- Introducing a unified Terraform-based open source tool for provisioning, deploying, installing, upgrading, and decommissioning DC/OS on AWS, GCP, and Azure.
+- Intuitive, streamlined installation with a quick start process - Spin up a DC/OS cluster with a few easy steps in 10-15 minutes. 
+- Officially recommended as a Mesosphere supported installation method with best practices built-in (i.e sequential masters & parallel agents in upgrade).
+- Restructured [Mesosphere installation documentation](https://docs.mesosphere.com/1.12/installing/evaluation/) to organize Mesosphere supported installation methods and Community supported installation methods.
+- Expanded DC/OS upgrade paths enable Mesosphere to skip specific [upgrade paths](https://docs.mesosphere.com/1.12/installing/production/upgrading/#supported-upgrade-paths) within a supported patch version of DC/OS (i.e upgrade from 1.11.1 => 1.11.5 in one move) and to skip upgrade paths between supported major to major versions of DC/OS (for example, enabling you to upgrade from 1.10.7 to 1.11.4 in one move).
```

Comment:
Need to change `1.11.1 => 1.11.5` & `upgrade from 1.10.7 to 1.11.4` to the 1.12 equivalent. 
@kevinmiller-conley can you please help here?

Reply:
Well, we don't have enough 1.12 versions to explain jumping minor updates in just 1.12 terms.

For jumping majors you could say 1.11.8 -> 1.12.1, but I think the versions that are in there explain it fine for now unless there's some reason it has to be 1.12?

Reply:
I think it would be okay either way unless we don't support a jump like 1.11.7 to 1.12.1 ....